controllers/articles.controller.js

```diff
@@ -22,10 +23,27 @@ exports.getArticles = (request, response, next) => {
     const { sort_by } = request.query
     const { order } = request.query
     const { topic } = request.query
+
+    
+    const promises = [selectTopics(), selectArticles(sort_by, order, topic)]
```

Comment:
There is some duplication here. You also `selectArticles` on line 42, and send a repsonse on line 38 and 44. I'd suggest refactoring this to remove one or the other - if your second promise (starting on line 42) resolves before the first, then the articles will be sent back without a topic check